examples/js/loaders/SVGLoader.js

```diff
@@ -1005,7 +1008,7 @@ THREE.SVGLoader.prototype = {
 
 		console.timeEnd( 'THREE.SVGLoader: Parse' );
 
-		return paths;
+		return [ paths, nodeMap ];
```

Comment:
Changing the return value of `parse()` is not backwards compatible again. Calling directly `SVGLoader.parse()` is a valid usage. It's not always necessary to use `SVGLoader.load()`. If you change the return type from an array of `ShapePath` to the proposed structure, user code will break.

It's up to @mrdoob to decide, if it's worth to change the loader in this way.